functions.d/ambient_match_icomera_wifi.fish

```diff
@@ -0,0 +1,41 @@
+function ambient_match_icomera_wifi
```

Comment:
This is cool!

But I also think it should probably be inlined in `icomera.fish`, just since it's not used anywhere else.